src/styles/components/_definition.scss

```diff
@@ -18,7 +20,7 @@ $breakpoint-tablet: 820px;
 
   @media only screen and (min-width: $breakpoint-tablet) {
     background:
-      linear-gradient(to right, white 50%, rgba(255 255 255 / 0%)),
+      linear-gradient(to right, var(--background-color) 50%, transparent),
```

Comment:
![Screen Shot 2022-10-31 at 9 38 45 AM](https://user-images.githubusercontent.com/43832671/199034798-4edf4459-1119-45a2-8ceb-e6b383e39c68.png)

If you want to get rid of this visual bug, you can just switch the gradient color to black on dark mode.

Reply:
Thanks for catching that Bennett! :) As for the Sparkbox logo--I'm not sure what is supposed to happen w/ that. I need to ask Jelly as in the design the logo is actually black on dark mode.